base/ca/src/com/netscape/ca/CRLIssuingPointConfig.java

```diff
@@ -293,4 +293,12 @@ public BigInteger getCRLEndSerialNo() throws EBaseException {
     public void setCRLEndSerialNo(BigInteger crlEndSerialNo) {
         putBigInteger("crlEndSerialNo", crlEndSerialNo);
     }
+
+    public boolean getScheduleUpdated()  throws EBaseException {
+        return getBoolean("scheduleUpdated",false);
+    }
+
+    public void setScheduleUpdated(Boolean updated) {
+        putBoolean("scheduleUpdated",updated);
```

Comment:
I think the config param name "scheduleUpdated" might be a bit confusing.  Would it make sense to use
ca.crl.MasterCRL.autoUpdateInterval.effectiveAtStart=true ?
if acceptable, might want to reflect the change of variable names in code just to be clear.

Reply:
I could do that.. but got anything more concise than that? :)